src/sections/2021/section-5.js

```diff
@@ -144,9 +144,9 @@ const Section5 = () => (
         </svg>
 
         <h3>Design systems are intended to cross complex boundaries, but realizing the full value of your investment is tricky work.</h3>
-        <p>Design systems are intended to cross complex boundaries, but realizing the full value of your investment is tricky work.</p>
+        <p>What do you need to support your design system?</p>
         <p>Take the Maturity Model Assessment to get feedback on where you are now, and suggestions that will help you move forward to benefit your whole organization.</p>
-        <a href="https://example.com" className="cmp-button cmp-button--inverse">Take the Survey</a>
+        <a href="https://example.com" className="cmp-button cmp-button--inverse">Take the <strong>Assessment</strong></a>
```

Comment:
Assessment shouldn’t be bolded here, please remove `<strong>` tags.

Reply:
ok! I assumed the bold on the doc meant to make the text bold. 
<img width="325" alt="Screen Shot 2021-07-08 at 9 34 17 AM" src="https://user-images.githubusercontent.com/24901036/124930864-ac7c8c80-dfcf-11eb-95dd-49cfb7c39507.png">

Reply:
Makes sense why you took it that way. That was to emphasize which word needed changed.